docs/source/examples/demo_deploy_cliptextmodel.ipynb

```diff
@@ -0,0 +1,343 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'\\n{\\n    \"name\": \"clip-vit-base-patch32\",\\n    \"version\": \\'1.0.0\\',\\n    \"model_format\": \"TORCH_SCRIPT\",\\n    \"model_config\": {\\n        \"model_type\": \"clip\",\\n        \"embedding_dimension\": 512,\\n        \"framework_type\": \"huggingface_transformers\"\\n    }\\n}\\n'"
+      ]
+     },
+     "execution_count": 57,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# config.json sample contents\n",
+    "\"\"\"\n",
+    "{\n",
+    "    \"name\": \"clip-vit-base-patch32\",\n",
+    "    \"version\": '1.0.0',\n",
+    "    \"model_format\": \"TORCH_SCRIPT\",\n",
+    "    \"model_config\": {\n",
+    "        \"model_type\": \"clip\",\n",
+    "        \"embedding_dimension\": 512,\n",
+    "        \"framework_type\": \"huggingface_transformers\"\n",
+    "    }\n",
+    "}\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Register model to OpenSearch\n",
+    "\n",
+    "* Model name in config.json should match .pt torchscript file name\n",
+    "* Record the model ID from the output of the next cell\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'ml_client' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[1;32mc:\\Users\\patri\\OneDrive\\Dahlia\\OpenSearch\\demo_deploy_cliptextmodel.ipynb Cell 11\u001b[0m line \u001b[0;36m4\n\u001b[0;32m      <a href='vscode-notebook-cell:/c%3A/Users/patri/OneDrive/Dahlia/OpenSearch/demo_deploy_cliptextmodel.ipynb#X13sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m model_path \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39m/fashion_clip_TS_v3.zip\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[0;32m      <a href='vscode-notebook-cell:/c%3A/Users/patri/OneDrive/Dahlia/OpenSearch/demo_deploy_cliptextmodel.ipynb#X13sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m model_config_path \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39m/config.json\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m----> <a href='vscode-notebook-cell:/c%3A/Users/patri/OneDrive/Dahlia/OpenSearch/demo_deploy_cliptextmodel.ipynb#X13sZmlsZQ%3D%3D?line=3'>4</a>\u001b[0m model_id_file_system \u001b[39m=\u001b[39m ml_client\u001b[39m.\u001b[39mregister_model(model_path, model_config_path, isVerbose\u001b[39m=\u001b[39m\u001b[39mTrue\u001b[39;00m, deploy_model \u001b[39m=\u001b[39m \u001b[39mFalse\u001b[39;00m)\n",
+      "\u001b[1;31mNameError\u001b[0m: name 'ml_client' is not defined"
+     ]
+    },
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
```

Comment:
Are we keeping this?

Reply:
No, this error text should be removed from the commit